### ISSUE TITLE
Pagination Aria Labels

### ISSUE THREAD

**Issue by luketa8** [grommet/grommet#7579]
[Pagination component](https://design-system.hpe.design/components/pagination) is failing 1.3.1, 1.3.6 and 4.1.2  [WCAG Rules](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.2&showtechniques=111%2C131%2C246%2C248%2C2410%2C1412#identify-purpose)

Current screen reader output for the “Items per page” dropdown is unclear.  There is no method to pass a label to the items per page Select step. The ARIA labeling lacks meaningful information and does not effectively convey the purpose of the component to assistive technologies.

It announces “Open drop,” which does not effectively convey the purpose of the control. Consider using a more descriptive and instructive label, such as “Items shown per page – 10 selected,” to better inform users of its function and current state.

**Comment by jcfilben** [comment]
@Sulaymon333 now that #7590 is merged can you update the Pagination rules 1.3.1, 1.3.6 and 4.1.2 on the design system site to passing?

**Comment by jcfilben** [comment]
@Sulaymon333 it looks like `pagination.stepLabel` is missing from the Grommet messages prop types and index.d.ts can we add this?
It will also need to be added to the documentation on grommet-site

**Comment by Sulaymon333** [comment]
@jcfilben I have fixed the missing prop types on both pagination component and on grommet sire
- Grommet Pagination PR - https://github.com/grommet/grommet/pull/7598
- Grommet site PR - https://github.com/grommet/grommet-site/pull/541
